outputs.tf

```diff
@@ -86,3 +86,25 @@ output "eventbridge_role_name" {
   description = "The name of the IAM role created for EventBridge"
   value       = try(aws_iam_role.eventbridge[0].name, "")
 }
+
+# EventBridge Pipes
+output "eventbridge_pipe_ids" {
```

Comment:
```suggestion
output "eventbridge_pipes_ids" {
```

Reply:
Trying to be consistent with the rest of the outputs. :)